test/pre-need/00-all-fields.e2e.spec.js

```diff
@@ -59,7 +59,7 @@ const runTest = E2eHelpers.createE2eTest(
       client.assert.cssClassPresent('.progress-bar-segmented div.progress-segment:nth-child(4)', 'progress-segment-complete');
       PageHelpers.completeBenefitSelection(client, testData.data.application);
       // do not run 'wcag2a' rules because of open aXe bug https://github.com/dequelabs/axe-core/issues/214
-      client.axeCheck('.main', { rules: ['section508'] })
+      client.axeCheck('.main', { rules: { 'aria-required-children': { enabled: false } } })
```

Comment:
I looks like we've been disabling entire rulesets rather than individual rules ([reference](https://www.axe-core.org/docs/#configure)).

Reply:
This is much better--thanks! There are other places we disable the whole set. Do you want to include finding and fixing them all in this PR or separately?

Reply:
Good idea, separate is better. I'll remove these changes.

Reply:
I'm not sure it's true we've been disabling rules in these tests. This `rules` array is being passed into our axe wrapper and supplied as part of the `run-only` configuration which seems consistent with the API docs:
https://github.com/department-of-veterans-affairs/vets-website/blob/master/test/util/nightwatch-commands/axeCheck.js#L26

So passing in `{ rules: ['section508'] }` would override the default `run-only` config, and not run the WCAG 2.0 rules.

I still think your idea to be more selective in what we disable is a great one, but maybe what should happen is we pass in an `exceptions` or `disable` array, and then in aXeCheck.js, add a `rules` value in addition to the `run-only`.